src/DetailsView/components/left-nav/fluent-side-nav.scss

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../../common/styles/common.scss';
+
+#side-nav-container {
```

Comment:
any reason you're using id here?

Reply:
So the panel provides an option to pass a host id as a prop, and if it is provided the panel will render inside this element, instead of a direct child of the body element, and I want this behavior to achieve some UX in spec.

Reply:
Great! That's awesome! 👍